docs/guide/service_container.md

```diff
@@ -1,46 +1,52 @@
-# Service Container
+# Service container
 
-## Definition
+A service container (aka Dependency Injection Container, DIC) is a special object that facilitates dependency resolution.
+It is based on the [dependency injection design pattern](http://en.wikipedia.org/wiki/Dependency_injection).
 
-A service container, (aka **DIC**, *Dependency Injection Container*) is a special object that greatly facilitates dependencies resolution in your application and sits on [Dependency Injection design pattern](http://en.wikipedia.org/wiki/Dependency_injection). Basically, this design pattern proposes to inject all needed objects and configuration into your business logic objects (aka **services**). It avoids the massive use of singletons, global variables or complicated factories and thus makes your code much more readable and testable. It avoids "magic."
+Dependency injection proposes to inject all needed objects and configuration into your business logic objects (known as **services**).
+It avoids the massive use of singletons, global variables or complicated factories and makes your code much more readable and testable.
 
-The main issue with dependency injection is how to resolve your dependencies for your services. This is where the service container comes into play. The role of a service container is to build and maintain your services and their dependencies. Basically, each time you need a service, you may ask the service container for it, which will either build it with the configuration you provided, or give you an existing instance if it is already available.
+The main issue with this pattern is how to resolve the dependencies for your services.
+This is where the service container comes into play. The role of a service container is to build and maintain your services and their dependencies.
+Each time you need a service, you may ask the service container for it.
+It will either build the service with the configuration you provided, or give you an existing instance if it is already available.
 
-## In eZ Platform
+eZ Platform uses the [Symfony service container](http://symfony.com/doc/master/book/service_container.html).
```

Comment:
Shouldn't it be specific Symfony version 2.8/3.4?
Same to all Symfony links from this PR